packages/@react-spectrum/overlays/src/Overlay.tsx

```diff
@@ -54,7 +54,7 @@ function Overlay(props: OverlayProps, ref: DOMRef<HTMLDivElement>) {
   }
 
   let contents = (
-    <Provider ref={ref} UNSAFE_style={{position: 'absolute', zIndex: 100000, top: 0, left: 0}}>
+    <Provider ref={ref} UNSAFE_style={{background: 'transparent'}}>
```

Comment:
background transparent? curious what this is for

Reply:
Provider has a background.

Reply:
Actually, maybe not needed anymore. I think it was when I had it absolute positioned over the whole page.

Reply:
Turns out it's needed otherwise Provider doesn't render a DOM node at all, and just acts as a context provider... We can revisit this.